sources/procfs.go

```diff
@@ -320,6 +325,14 @@ func (s *lustreProcfsSource) generateClientMetricTemplates() error {
 			{"stats", "inode_permission_total", inodePermissionHelp, s.counterMetric},
 			{"xattr_cache", "xattr_cache_enabled", "Returns '1' if extended attribute cache is enabled", s.gaugeMetric},
 		},
+		"mdc/*": {
+			{"rpc_stats", "rpc_in_flight", rpcsInFlightHelp, s.gaugeMetric},
```

Comment:
well, rpcs_in_flight would cause metric conflicts, wouldn't it? Or can mdc and osc not be populated at the same time? What do the labels end up looking like for those metrics?